### ISSUE TITLE
EffectiveTldFinderTests failing after merge pull request #100

### ISSUE THREAD

**Issue by aecio** [crawler-commons/crawler-commons#103]
Although maven build is finished successfully, the two tests cases in class `EffectiveTldFinderTest` seems to be failing after pull request #100. Can anyone reproduce this?

```
End of Report:

Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.141 sec

Results :

Failed tests:   testWildcardEtld(crawlercommons.url.EffectiveTldFinderTest)
  testExceptionEtld(crawlercommons.url.EffectiveTldFinderTest)

Tests run: 92, Failures: 2, Errors: 0, Skipped: 0

[ERROR] There are test failures.

Please refer to ~/workspace/crawler-commons-clean/target/surefire-reports for the individual test results.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.965 s
[INFO] Finished at: 2015-10-07T00:13:01-04:00
[INFO] Final Memory: 27M/794M
[INFO] ------------------------------------------------------------------------
```

**Comment by jnioche** [comment]
thanks @aecio I'll investigate this

**Comment by jnioche** [comment]
OK, fixed in [b7ccc8d]
This was due to changes to the UK section of the reference list. Thanks for reporting it
